### PR TITLE
Enable Portis wallet

### LINKED ISSUE #434: [Wallet] Enable Portis
**Description**
Enable Portis

The wallet was disabled:
image

We would like to enable and test that it works as expected.
It's already working. We just needs to make sure the codes are OK for all our environments and ENS.

**Additional context**
It was not well tested, so we removed all wallets except metamask, so we can added and test them in isolation. See https://github.com/gnosis/gp-swap-ui/pull/430

### PR DESCRIPTION
**Pull request by henrypalacios**
closes #434
Enable the ability to connect via `Portis`

## To Test
- Go to the [**Connect to a wallet**](https://pr2404--gpswapui.review.gnosisdev.com/) button and click on the `Portis` option.

## Background

*This PR is part of the activation of previously supported wallets #2165*

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @henrypalacios , great job!
I was able to run wrap/unwrap/approve transactions, place Buy/sell orders using Portis wallet. 

Some issues I faced: 
1. When logout from all accounts in the Portis wallet, the app is not auto-disconnected. I have added this note to the #1974 issue. 
log out

2. It would be nice to update the app icon when connecting to Portis: 
icon 2
icon 3
icon
to the actual one: 
image
3. And the last very interesting case: the Portis wallet has 'Trust this app' toggle, that allows to auto-sign orders 
image
So when I switch this toggle on, my sell/buy order is auto-signed without any manual action, as this transaction is considered as a gasless and does not exceed 1$ in fees. But in the end a signed and traded order might cost more than 1$. 
My question is: can we do anything with it from our side and force a user always manually sign orders in Portis wallet with no regard to this setting? 
@alfetopito , WDYT? 

And just to mention, as this might not be related to this PR: I was able to catch a 'Free!' transaction in this PR. 
free

**Comment by alfetopito** [comment]
Not sure what I'm doing wrong, but I'm stuck in a login loop:

https://user-images.githubusercontent.com/43217/153077142-b6c23182-ddb8-4623-b33c-6287b12792bf.mov

File: .env.production [hunk 1]
```diff
@@ -29,7 +29,7 @@ REACT_APP_NETWORK_URL_4=https://rinkeby.infura.io/v3/1221fd11e90849509afafd330ec
 REACT_APP_NETWORK_URL_100=https://rpc.xdaichain.com
 
 # Wallets
-REACT_APP_PORTIS_ID="c0e2bf01-4b08-4fd5-ac7b-8e26b58cd236"
+REACT_APP_PORTIS_ID="c591a488-8b86-41e1-8d2e-f4b169efd3aa"
```
Comment (alfetopito) [inline .env.production:32]: That's the latest 👍

**Comment by alfetopito** [comment]
@elena-zh I have update the app settings on Portis
Screen Shot 2022-02-08 at 13 40 10

Using the latest logo, updated the domains and disabled "auto" sign messages.
The last change should take care of the undesired behaviour you noticed.

Regarding the free tx, as I said, might have been due to momentary cheap gas prices.
Place tx next time you see it so we have a reference to check with backend whether this is a bug or not.

**Comment by elena-zh** [comment]
Hey @henrypalacios , I have just found that the issue @alfetopito reported above https://github.com/gnosis/cowswap/pull/2404#issuecomment-1033070430 is a browser-specific and is reproducible in Firefox. 
See the video: https://watch.screencastify.com/v/6vllCR5cGPqCTEw1J3NW

**Comment by alfetopito** [comment]
> Hey @henrypalacios , I have just found that the issue @alfetopito reported above [#2404 (comment)](https://github.com/gnosis/cowswap/pull/2404#issuecomment-1033070430) is a browser-specific and is reproducible in **Firefox**. See the video: https://watch.screencastify.com/v/6vllCR5cGPqCTEw1J3NW
> 
> The same issue is reproducible in **Brave** browser.
> 
> So I can easily connect the wallet in Chrome only.

The same happens on Uniswap and the Safe, so...

I suggest we reach out to Portis as this might be on their side.
Don't think there will be much we can do on ours.

**Comment by henrypalacios** [comment]
>The same happens on Uniswap and the Safe, so...
>
>I suggest we reach out to Portis as this might be on their side.
>Don't think there will be much we can do on ours.

I want to add that it only happened to me with Brave V1.35.101, I had no problems with firefox v96.0 (neither Uniswap, gnosis-safe).

**Pull request closed without merging**